### DISCUSSION TITLE
Not yet support xapk

### DISCUSSION THREAD

**Discussion by trandua** [iBotPeaches/Apktool#3157] [Ideas]
Not yet support decompile xapk file :(

**Comment by giacomoferretti** [comment]
XAPK files are simply ZIP files containing the base apk and the various split apks.

**Comment by iBotPeaches** [comment]
I'll use this ticket as the backlogged feature request. Maybe at some point Apktool could reach into the xapk and pull out what it needs, but that gets really confusing with all the splits/archs that may exist.

**Comment by MrIkso** [comment]
> I'll use this ticket as the backlogged feature request. Maybe at some point Apktool could reach into the xapk and pull out what it needs, but that gets really confusing with all the splits/archs that may exist.

Very soon there will be such applications.  I already thought about it, it is possible to unpack only the basic arch from this archive, and at assembly to pack everything back.  also to add a key whether to unpack all split apk from archive.  I personally have such an idea

**Comment by thiagobenine** [comment]
Any updates on this ?
